tokio/tests/rt_unstable_metrics.rs

```diff
@@ -170,6 +170,26 @@ fn worker_park_count() {
     assert!(1 <= metrics.worker_park_count(1));
 }
 
+#[test]
+fn worker_park_unpark_count() {
+    let rt = current_thread();
+    let metrics = rt.metrics();
+    rt.block_on(async {
+        time::sleep(Duration::from_millis(1)).await;
+    });
+    drop(rt);
+    assert!(2 <= metrics.worker_park_unpark_count(0));
+
+    let rt = threaded();
+    let metrics = rt.metrics();
+    rt.block_on(async {
+        time::sleep(Duration::from_millis(1)).await;
+    });
+    drop(rt);
+    assert!(2 <= metrics.worker_park_unpark_count(0));
+    assert!(2 <= metrics.worker_park_unpark_count(1));
```

Comment:
It's not clear to me why both workers should wake up. One of them has to so that it can wake the timer, but I don't think the other thread is guaranteed to wake up.

Reply:
You are correct. I changed it accordingly.